mozdownload/cli.py

```diff
@@ -163,7 +167,10 @@ def cli(argv=None):
             scraper_type = 'direct'
 
         build = factory.FactoryScraper(scraper_type, **kwargs)
-        build.download()
+        if kwargs.get('print_url'):
+            print build.url
```

Comment:
Instead of `print` you should use a call to `logger.info()` so that it get printed correctly.